zips/zip-0234.md

```diff
@@ -81,87 +105,96 @@ future block subsidies.
+approach, issuance will jump _up_ some amount at activation. This amount
+should be equivalent to all ZEC burnt prior to that height times
+$\mathtt{BLOCK\_SUBSIDY\_FRACTION}$. For example, if a total of 100,000 ZEC
+were burnt prior to the pre-defined constant activation height, then at
+activation the issuance would be larger than BTC-style issuance by
+$100\_000\text{ ZEC} \cdot \mathtt{BLOCK\_SUBSIDY\_FRACTION}\!$,
+which we calculate equals $0.04126$ ZEC. This example is chosen to
+demonstrate that a very large burn amount (much larger than expected) would
+elevate issuance by a relatively small amount. For this reason, we believe
+a pre-defined constant is a better approach to achieving Key Objective 6
+than a "dynamic latch" logic because it is so much simpler to implement
+and reason about.
+
+$\mathsf{MoneyReserveAfter}(\mathsf{height}) \;\,=$ The value of the Money Reserve
+after the specified block height.
 
 ## Issuance Calculation
 
 At the `DEPLOYMENT_BLOCK_HEIGHT`, nodes should switch from the current issuance
 calculation, to the following:
 
-`BlockSubsidy(block_height) = ceiling(BLOCK_SUBSIDY_FRACTION * MoneyReserveAfter(block_height - 1))`
+$\mathsf{BlockSubsidy}(\mathsf{height}) = \mathsf{ceiling}(\mathtt{BLOCK\_SUBSIDY\_FRACTION} \cdot \mathsf{MoneyReserveAfter}(\mathsf{height} - 1))$
+
+## Applicability
+
+All of these changes apply identically to Mainnet and Testnet.
+
 
 # Rationale
 
 * Using an exponential decay function satisfies **Requirements 1** and **2** above.
 * We round up to the next zatoshi to satisfy **Requirement 3** above.
 
-## `BLOCK_SUBSIDY_FRACTION`
+## BLOCK_SUBSIDY_FRACTION
 
-Let `IntendedMoneyReserveFractionRemainingAfterFourYears` = 0.5.
+Let $\mathsf{IntendedMoneyReserveFractionRemainingAfterFourYears} = 0.5\!$.
 
-The value `4126 / 10_000_000_000` satisfies the approximation within +0.002%:
+The value $4126 / 10\_000\_000\_000$ satisfies the approximation within $\pm 0.002\%$:
 
-`(1 - BLOCK_SUBSIDY_FRACTION)^PostBlossomHalvingInterval ≈ IntendedMoneyReserveFractionRemainingAfterFourYears`
+$(1 - \mathtt{BLOCK\_SUBSIDY\_FRACTION})^\mathsf{PostBlossomHalvingInterval} \approx \mathsf{IntendedMoneyReserveFractionRemainingAfterFourYears}$
 
-Meaning after a period of 4 years around half of Money Reserve will be issued as
-block subsidies, thus satisfying **Requirement 4**.
+This implies that after a period of 4 years around half of Money Reserve will
+have been issued as block subsidies, thus satisfying **Requirement 4**.
 
-The largest possible value in the Money Reserve is `MAX_MONEY`, in the
+The largest possible value in the Money Reserve is $\mathsf{MAX\_MONEY}$, in the
 theoretically possible case that all issued funds are burned. If this happened,
-the largest interim sum in the block subsidy calculation would be `MAX_MONEY *
-4126 / 10000000000`.
+the largest interim sum in the block subsidy calculation would be
+$\mathsf{MAX\_MONEY} \cdot 4126 / 10\_000\_000\_000\!$.
```

Comment:
Same question here; does this render properly? I thought that the `$...$` syntax was ReStructuredText only.

Reply:
It's the opposite, reStructuredText requires 
```
:math:`...`
```

Reply:
Actually we should maybe apply a preprocessor so that we can use `$...$` in rst as well. The `:math:` thing is just silly.